site/website/blog/2020-07-16-designing-evergreen-cache-cdc.md

```diff
@@ -206,9 +206,9 @@ data, depending on the underlying disk technology.
 [Hazelcast Jet](https://jet-start.sh/) is a distributed stream
 processing framework built on Hazelcast and combines a cache with
 fault-tolerant data processing.
-It has sources and sinks to integrate with several file, messaging and
-database systems (such as Amazon S3, Kafka, message brokers and
-relational databases).
+It has sources and sinks to integrate
```

Comment:
Is this some leftover change? It doesn't seem to improve anything.

Reply:
Yes, leftover, fixed it.